README.md

```diff
@@ -188,6 +188,19 @@ const MyEvents = {
 };
 ```
 
+### Auto-reconnection
+
+In order to re-establish a lost connection when triggering an event a `Channel`
+needs to implement the `autoReconnect` method.
```

Comment:
Can you link to the channel example as well?

Reply:
fixed